### PR TITLE
topology2: add dmic pipeline to nocodec topology

### PR DESCRIPTION
**Pull request by yongzhi1**
With dmic definition already in, add pipeline to make use of the pcm.

Signed-off-by: Yong Zhi <yong.zhi@intel.com>

### CODE REVIEW

**Comment by lgirdwood** [comment]
@ranj063 @juimonen can you guys add yourself as codeowners for topology 2. Thanks !

**Comment by ranj063** [comment]
@lgirdwood the patch looks ok but we need @RanderWang or @bardliao to test the topology before merging.

**Review by RanderWang — APPROVED** [review]
Tested on TGL

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**
